website/docs/cli-reference.md

```diff
@@ -123,6 +123,44 @@ An optional salt controlling where the contract will be deployed. The contract d
 Used by whitelisted users for deploying contracts in Alpha MainNet.
 #### `--wait-for-acceptance`
 Waits for transaction to be accepted on chain.
+### `deploy-account`
+Sends deploy-account transaction. The account contract must be already declared and prefunded.
```

Comment:
```suggestion
Deploys a new account on selected network. The account contract must be already declared and prefunded.
```

Reply:
I think the original version is more correct. Deployer deploys the account contract, not Protostar.